nixos/modules/hardware/ryzen-smu.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.ryzen_smu;
+  ryzen_smu = config.boot.kernelPackages.ryzen_smu;
```

Comment:
Does `ryzenadj` use `ryzen_smu` at all?
Does `ryzen_smu` work with laptop CPUs?
 I put this PR together because `RyzenAdj` refused to work with my desktop AMD CPU.

Reply:
> I would like to add a warn for users installing `ryzenadj`

What's your point of view @rhendric ? I'm sorry to bother you, I thought you might be interested in this PR as `ryzenadj` 's maintainer.

Reply:
I don't have a great picture of all the CPUs that are and are not supported by all the different tweaking utilities. I use RyzenAdj on a 5700U and I don't seem to need any modules to enable its full capabilities there (I don't have any `nan`s in my `ryzenadj -i` output), so this warning wouldn't seem to be applicable to everyone.

Reply:
I know RyzenAdj only supports laptop CPUs (with a "U"?), so desktop CPU users can't use it. For example, my 5900x is not supported by RyzenAdj, as it is a desktop chip.

I am fairly sure RyzenAdj does not use `ryzen_smu` at all, [there is no reference to libsmu in the repo](https://github.com/search?q=repo%3AFlyGoat%2FRyzenAdj%20libsmu&type=code). RyzenAdj has it's own smu library: `ryzen_nb_smu`.

Rather than warning users when they happen to have one package installed, we probably need a wiki to recommend packages users can try. [The arch wiki is actually pretty good](https://wiki.archlinux.org/title/Ryzen), it mentions both RyzenAdj and ryzen_smu, and led me to a number of other ryzen utilities.

Reply:
> Rather than warning users when they happen to have one package installed, we probably need a wiki to recommend packages users can try. [The arch wiki is actually pretty good](https://wiki.archlinux.org/title/Ryzen), it mentions both RyzenAdj and ryzen_smu, and led me to a number of other ryzen utilities.

I agree  with it now.